### ISSUE TITLE
Open the File Explorer when workspace is loaded

### ISSUE THREAD

**Issue by l0rd** [eclipse-che/che#17969]
### Is your enhancement related to a problem? Please describe.

When I click on a "Getting Started" I see this:

<img width="782" alt="image" src="https://user-images.githubusercontent.com/606959/94274742-58278500-ff46-11ea-85d3-82672ee26cce.png">

As a first time user I may think that no source code has been cloned yet and that's also because the actions that are proposed are "New File..." and "Git Clone...".

### Describe the solution you'd like

I would like the file explorer to be opened as soon as the workspace is loaded AND that the project folder is expanded (first level only). 

I would also like that the README.md is loaded instead of the welcome page, but this is a separate issue.

<img width="720" alt="image" src="https://user-images.githubusercontent.com/606959/94275539-61652180-ff47-11ea-8267-89ede796f086.png">

**Comment by azatsarynnyy** [comment]
All enhancements are done by the PR https://github.com/eclipse/che-theia/pull/885:
- each time when a new Workspace started, Project Explorer is opened automatically
- fixed opening `README.md` file - Che Theia starts looking for a file only after finishing cloning the project's sources
- project folder that contains a `README.md` file is auto-expanded

![Peek 2020-10-27 10-07](https://user-images.githubusercontent.com/1636395/97275711-f0af6e80-183e-11eb-8849-200b1a25c564.gif)
